commit 6918e2cde9b03c084be49e450b04d96340707b30 (refs/changes/71/4571/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:16:13+00:00 (1 year, 2 months ago)
    
    TRITON-606 ur-agent should be aware of alternate admin nictags (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 371c404..2daffec 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-ur-agent",
   "description": "Ur",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "author": "Joyent (joyent.com)",
   "main": "./ur-agent",
   "private": true,
